src/components/ContactUs/index.js

```diff
@@ -0,0 +1,194 @@
+                      {errors.lastName && touched.lastName && (
+                        <ErrorMsg>{errors.lastName}</ErrorMsg>
+                      )}
+                    </td>
+                  </tr>
+                  <tr>
+                    <td>
+                      <label htmlFor="phone">Phone:</label>
+                      <ContactInput
+                        type="text"
+                        id="phone"
+                        placeholder="(123) 123-4567"
+                        name="phone"
+                        onChange={handleChange}
+                        onBlur={handleBlur}
+                        value={values.phone}
+                      />
+                      {errors.phone && touched.phone && (
+                        <ErrorMsg>{errors.phone}</ErrorMsg>
+                      )}
+                    </td>
+                    <td>
+                      <label htmlFor="email">Email:</label>
+                      <ContactInput
+                        type="text"
+                        id="email"
+                        placeholder="jane-doe@example.com"
+                        name="email"
+                        value={values.email}
+                        onChange={handleChange}
+                        onBlur={handleBlur}
+                      />
+                      {errors.email && touched.email && (
+                        <ErrorMsg>{errors.email}</ErrorMsg>
+                      )}
+                    </td>
+                  </tr>
+                </tbody>
+              </ContactUsTable>
+              <label htmlFor="message">Message:</label>
+              <ContactTextArea
+                type="text"
+                id="message"
+                onChange={handleChange}
+                name="msg"
+                row="5"
+                value={values.msg}
+                onBlur={handleBlur}
+              />
+              {errors.msg && touched.msg && <ErrorMsg>{errors.msg}</ErrorMsg>}
+              <br />
+              <ContactUsBtn type="submit">Submit</ContactUsBtn>
+              <ContactCloseBtn onClick={props.click}>Exit</ContactCloseBtn>
+            </ContactForm>
+          </div>
+        )}
+      </ContactFormContainer>
+    </ContactBackground>
+  );
+};
+
+const EnhancedWithFormik = withFormik({
+  mapPropsToValues: () => ({
+    firstName: "",
+    lastName: "",
+    phone: "",
+    email: "",
+    msg: ""
+  }),
+
+  validate: values => {
+    const errors = {};
+
+    if (!values.email) {
+      errors.email = "Required";
+    } else if (
+      !/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values.email)
+    ) {
+      errors.email = "Invalid email address";
+    }
+
+    if (!values.firstName) {
+      errors.firstName = "Required";
+    }
+
+    if (!values.lastName) {
+      errors.lastName = "Required";
+    }
+
+    if (!values.phone) {
+      errors.phone = "Required";
+    } else if (!/^\D?(\d{3})\D?\D?(\d{3})\D?(\d{4})$/.test(values.phone)) {
```

Comment:
it is nice to assign regexes to named variables so it is easier to grok its intent ie: `const isPhoneNumber = /^\D?(\d{3})\D?\D?(\d{3})\D?(\d{4})$/.`  ... `else if (!isPhoneNumber.test(values.phone)) {`